FrostySdk/Ebx/TypeRef.cs

```diff
@@ -23,6 +23,16 @@ public TypeRef(Guid guid)
         {
             typeGuid = guid;
             typeName = TypeLibrary.Reflection.LookupType(guid);
+            // type is likely using a signed GUID
+            if (typeName == "" || typeName == guid.ToString())
```

Comment:
```suggestion
            if (typeName == guid.ToString())
```
else it will crash when generating the type info cache